app/org/sagebionetworks/bridge/hibernate/HibernateAccountConsent.java

```diff
@@ -0,0 +1,82 @@
+package org.sagebionetworks.bridge.hibernate;
+
+import javax.persistence.Embeddable;
+
+/**
+ * Consent, as represented by Hibernate. Note that this is different from the ConsentSignature class because of how
+ * Hibernate marshalls data to and from the database. In particular, it doesn't contain any of the keys (account ID,
+ * subpopulation GUID, signedOn). This is because Hibernate doesn't allow us to have table keys in an embeddable'
+ * object.
+ */
```

Comment:
The user can withdraw and sign the consent again, so using consentCreatedOn as the primary key could result in conflict. The signedOn timestamp uniquely identifies each consent each time the user signs it.

As for why it can't be in the consent object, this is a limitation of Hibernate.